src/bokeh/models/ui/menus.py

```diff
@@ -102,23 +112,22 @@ def __init__(self, *args, **kwargs) -> None:
     An optional action (callback) associated with this item.
     """)
 
-class CheckableItem(ActionItem):
-    """ A two state checkable menu item. """
+class ActionItem(MenuItem):
 
     # explicit __init__ to support Init signatures
     def __init__(self, *args, **kwargs) -> None:
+        deprecated((3, 7, 0), "ActionItem", "MenuItem")
         super().__init__(*args, **kwargs)
 
-    checked = Bool(default=False, help="""
-    The state of the checkable item.
-
-    Checked item is represented with a tick mark on the left hand side
-    of an item. Unchecked item is represented with an empty space.
-    """)
+class CheckableItem(ActionItem):
+    """ A two state checkable menu item. """
 
-    # TODO group = Either(Instance(MenuGroup), Auto)
+    # explicit __init__ to support Init signatures
+    def __init__(self, *args, **kwargs) -> None:
+        deprecated((3, 7, 0), "CheckableItem", "ActionItem.checked")
```

Comment:
Isn't `ActionItem` above also deprecated?

Reply:
Yes, I should have changed the base class to `MenuItem`.